test/spec/api/createVis/create-vis.spec.js

```diff
@@ -0,0 +1,344 @@
+      expect(visModel.get('interactiveFeatures')).toEqual(false);
+    });
+    it('should display the "loader" overlay by default [loaderControl, tiles_loader]', function () {
+      // loaderControl and tiles_loader appear to do the same.
+      var visJson = scenarios.load(0);
+      var visModel = createVis(this.containerId, visJson);
+      expect(visModel.overlaysCollection.findWhere({ type: 'loader' })).toBeDefined();
+    });
+    it('should display the "logo" by default', function () {
+      var visJson = scenarios.load(0);
+      var visModel = createVis(this.containerId, visJson);
+      expect(visModel.overlaysCollection.findWhere({ type: 'logo' })).toBeDefined();
+    });
+    it('should not display empty infowindow fields by default', function () {
+      var visJson = scenarios.load(0);
+      var visModel = createVis(this.containerId, visJson);
+      expect(visModel.get('showEmptyInfowindowFields')).toEqual(false);
+    });
+    it('should have legends', function () {
+      var visJson = scenarios.load(0);
+      var visModel = createVis(this.containerId, visJson);
+      expect(visModel.settings.get('showLegends')).toEqual(true);
+    });
+    it('should show layer selector', function () {
+      var visJson = scenarios.load(0);
+      var visModel = createVis(this.containerId, visJson);
+      expect(visModel.settings.get('showLayerSelector')).toEqual(true);
+      expect(visModel.settings.get('layerSelectorEnabled')).toEqual(true);
+    });
+    it('should allow scrollwheel by default', function () {
+      var visJson = scenarios.load(0);
+      var visModel = createVis(this.containerId, visJson);
+      expect(visModel.map.get('scrollwheel')).toEqual(true);
+    });
+    it('should have "infowindow" enabled by default', function () {
+      pending('It seems that this option is no longer being used');
+    });
+    it('should have "tooltip" by default', function () {
+      pending('It seems that this option  is no longer being used');
+    });
+  });
+
+  describe('Options', function () {
+    describe('skipMapInstantiation', function () {
+      it('should instantiate map when skipMapInstantiation option is falsy', function (done) {
+        var visJson = scenarios.load(0);
+        var visModel = createVis(this.containerId, visJson);
+        setTimeout(function () {
+          expect(visModel._instantiateMapWasCalled).toEqual(true);
+          done();
+        }, 25);
+      });
+      it('should NOT instantiate map when skipMapInstantiation option is truthy', function (done) {
+        var visJson = scenarios.load(0);
+        var visModel = createVis(this.containerId, visJson, { skipMapInstantiation: true });
+        setTimeout(function () {
+          expect(visModel._instantiateMapWasCalled).toEqual(false);
+          done();
+        }, 25);
+      });
+    });
+  });
+
+  describe('VisModel.map', function () {
+    it('should have the right title', function () {
```

Comment:
Isn't this very similar to: https://github.com/CartoDB/cartodb.js/pull/1740/files#diff-5707dcdc5d2b4ea40894b5c990ab7c97R58?

Reply:
Same for some of 👇